addons/l10n_dk/data/account_account_types.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!--
+     (c) 2018 Odoo House ApS
+     <field name="type">receivable</field>
+     <field name="type">payable</field>
+     <field name="type">liquidity</field>
+     <field name="type">other</field>
+    -->
+    <data noupdate="True">
```

Comment:
Are you sure all these new types are necessary? Some of them seems to be duplicates of types already defined in https://github.com/odoo/odoo/blob/master/addons/account/data/data_account_type.xml

For example:

l10n_dk.account_type_revenue == account.data_account_type_revenue?
l10n_dk.account_type_prepayments == account.data_account_type_prepayments?
l10n_dk.account_type_direct_costs == account.data_account_type_direct_costs?
l10n_dk.account_type_depreciation == account.data_account_type_depreciation?